properties.md

```diff
@@ -9,13 +9,13 @@ This document is meant to provide an exhaustive list of properties that can be a
 | Key   | Semantics | Type     | Values    | Reference |
 | ----- | --------- | -------- | --------- | --------- |
 | [`contains`](/profiles/epub.md#contains)  | Indentifies content contained in the linked resource, that cannot be strictly identified using a media type.  | Array  | `mathml`, `onix`, `remote-resources`, `js`, `svg` or `xmp`  | [EPUB Profile](/profiles/epub.md#properties) |
-| [`encrypted`](/profiles/epub.md#encrypted)  | Indicates that a resource is encrypted/obfuscated and provides relevant information for decryption.  | [Encryption Object](/profiles/epub.md#encrypted)  | See the definition for the [Encryption Object](/profiles/epub.md#encrypted) | [EPUB Profile](/profiles/epub.md#properties) |
 | [`layout`](/profiles/epub.md#layout)  | Hint about the nature of the layout for the linked resources.  | String  | `fixed` or `reflowable`  | [EPUB Profile](/profiles/epub.md#properties) |
 | [`media-overlay`](/profiles/epub.md#media-overlay)  | Location of a media-overlay for the resource referenced in the Link Object.  | URI  | Any valid relative or absolute URI  | [EPUB Profile](/profiles/epub.md#properties) |
```

Comment:
Since you're working on the EPUB profile, shouldn't be media-overlay property removed? If I remember correctly, we decided to use `alternate` instead.

Reply:
It's not the purpose of this branch, but ok let's clean this here before working on media-overlays differently.